Libraries/HyTech_CAN/CAN_ID.h

```diff
@@ -55,9 +55,8 @@
 #define ID_CHARGER_CONTROL                                          0x1806E5F4
 #define ID_CHARGER_DATA                                             0x18FF50E5
 
-#define ID_ACU_SHUNT_MEASUREMENT                                    0x401
-#define ID_EM_STATUS                                                0x400
-#define ID_EM_MEASUREMENT                                           0x100
```

Comment:
Don't we need these for coulomb counting?

Reply:
Yeah, these got broken in the process of fixing some other diff issues, fixed now